meson.build

```diff
@@ -538,12 +538,31 @@ configure_file(input : 'dbus/org.freedesktop.ratbag1.conf.in',
 
 #### tools ####
 
+dep_python3 = dependency('', required : false)
```

Comment:
`''` is this a typo?

Reply:
no, this is the idiomatic "not found" dependency. This creates a dependency object that will never be found, but allows you to make sure that dep_python3 is always a dependency object

Reply:
I've added a comment explaining this, and also opened an upstream documentation addition about this: https://github.com/mesonbuild/meson/pull/7122

Reply:
Okay, I still need to test but this looks fine. Please fix the indentation, you are using tabs while we use 2 spaces.

Reply:
The latest version should be using spaces. I got confused because the meson.build has both tabs and spaces in it.